EzMap.Domain/Models/PoiConfiguration.cs

```diff
@@ -12,5 +13,14 @@ public void Configure(EntityTypeBuilder<Poi> builder)
         builder.Property(p => p.Name);
         builder.HasOne(p => p.User).WithMany(u => u.SelectedPois).HasForeignKey(p => p.UserId);
         builder.HasQueryFilter(p => p.DeletedDate == null);
+
+        builder.HasMany<Collection>(x => x.ListOfPois)
+            .WithMany(x => x.Pois)
+            .UsingEntity(
+                "PoiCollection",
+                l => l.HasOne(typeof(Collection)).WithMany().HasForeignKey("CollectionsId").HasPrincipalKey(nameof(Collection.Id)),
```

Comment:
This is okay, but not optimal. See warning [here](https://learn.microsoft.com/en-us/ef/core/modeling/relationships/many-to-many#:~:text=please%20don't%20attempt%20to%20fully%20configure%20everything%20even%20when%20it%20is%20not%20needed). We should not attempt to explicitly map everything (especially for entities that we don't even use in code), the code gets really confusing really quickly.

In the end, we should strive for the minimal amount of code that fully expresses our intent, not most amount of code that is barely readable.

Also these hardcoded string names are terribly bad (`CollectionsId` with an S, PoisId with an S as well...)

Reply:
Microsoft told me that :(

Reply:
Nope MS told u otherwise, you just read it half-heartedly, please see the link I sent